oshi-core/src/main/java/oshi/hardware/NetworkIF.java

```diff
@@ -61,6 +61,13 @@
      */
     String getDisplayName();
 
+    /**
+     * Interface alias.
```

Comment:
Let's me more verbose and describe the IfAlias per the RFC so it's clear what we're talking about.